docs/resources/instance_server.md

```diff
@@ -265,7 +271,10 @@ In addition to all above arguments, the following attributes are exported:
 - `root_volume`
     - `volume_id` - The volume ID of the root volume of the server.
 - `private_ip` - The Scaleway internal IP address of the server.
-- `public_ip` - The public IPv4 address of the server.
+- `public_ip` - The public IP address of the server.
```

Comment:
Precise if it is going to be a v4 or v6 address

Reply:
It will be either v4 or v6 depending of the IP type